sway-core/src/optimize.rs

```diff
@@ -2518,7 +2529,7 @@ mod tests {
         let output = path_converter.replace_all(output.as_str(), "$1/path/to/$2");
 
         if output != expected {
-            tracing::error!("{}", prettydiff::diff_lines(&expected, &output));
+            println!("{}", prettydiff::diff_lines(&expected, &output));
```

Comment:
@kayagokalp I've had to do this here similarly to https://github.com/FuelLabs/sway/pull/1624 to be able to see the actual diffs when I was building the tests.

Reply:
I think we should go like that 👍. I will look into places we had to come back to `println!` and think about what we can do about those places (or whether we should do something for them)